tests/integration/components/volunteer-headshot-test.js

```diff
@@ -0,0 +1,63 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import volunteerHeadshot from '../../pages/components/volunteer-headshot';
+
+let page = PageObject.create(volunteerHeadshot);
+
+const userRoles = [
+  { role: { name: 'Developer' } },
+  { role: { name: 'Ember Developer' } },
+  { role: { name: 'UX Designer' } },
+  { role: { name: 'Software Engineer' } },
+  { role: { name: 'Project Coordinator' } },
+  { role: { name: 'Designer & Developer' } }
+];
+
+moduleForComponent('volunteer-headshot', 'Integration | Component | volunteer headshot', {
+  integration: true,
+  beforeEach() {
+    this.set('user', {
+      name: 'Test User',
+      photoThumbUrl: 'http://fillmurray.com/200/200',
```

Comment:
@JoshSmith that's a good idea... but no more great photos of Bill Murray in tests.